README.md

```diff
@@ -7,130 +7,76 @@
 
-```
-├── Cargo.toml
-├── Makefile.toml
-├── index.html
-└── src
-    └── lib.rs
-```
 
 We currently have two template repositories:
 - [Quickstart](https://github.com/seed-rs/seed-quickstart)
 - [Webpack quickstart](https://github.com/seed-rs/seed-quickstart-webpack)
 
-# Why Use Seed
+The [examples](examples/) are another good resource.
 
-### Rust
-You may prefer writing in Rust and appreciate its benefits, including:
-- Rust **safety**.
-- Rust **compile-time error, type, and immutability checking**.
-- Rust built-in testing.
-- Rust speed.
-- Cleaner code and less runtime errors.
-- Cargo packages.
-- Potential pipeline/stack language consistency.
-
-### Developer Experience
-Our main focus is on developer experience, the benefits of which are currently:
-- Seed has a *batteries-included* approach, meaning less boilerplate and dependencies.
-- Macro syntax removes the need for transpiling and integrates naturally and flexibly with the language. This also means all the pains of embedded HTML are avoided; linting, commenting, etc. all work out of the box.
-- Built in elm-like architecture, no need for another state manager.
-- Perhaps you find JS frameworks and tools difficult to set up.
-- Very active development.
-
-See more on our [about](https://seed-rs.org/guide/about) page.
-
-# Why Not Use Seed
-- Seed is still under development, so there may be breaking changes at times. However, Seed is more than stable enough for personal projects, and production apps are in development.
-- Finding Rust/WASM/Seed help outside of [Discord](https://discord.gg/JHHcHp5) may be difficult, as tutorials and guides outside the official ones aren't yet prevalent.
-- Seed doesn't have as many existing reusable components that more mature frameworks have (date-pickers, etc.), so you may need to implement them yourself, or adapt them from existing solutions.
-
-# Documentation
-- Guides on [seed-rs.org](https://seed-rs.org)
-- API documentation on [docs.rs/seed](https://docs.rs/seed)
-
-# Resources
+## FAQ
+### How stable is Seed?
+It should be stable enough for personal projects. There are some using it in production. However, keep in mind Seed is still growing and improving and the framework is changing.
+
+## Documentation
+- Guides can be found at [seed-rs.org](https://seed-rs.org)
+- API documentation can be found at [docs.rs/seed](https://docs.rs/seed)
+
+## Resources
 ### Seed
 - [Awesome-seed-rs](https://github.com/seed-rs/awesome-seed-rs): A curated list of resources
-- [Seed Realworld](https://github.com/seed-rs/seed-rs-realworld): A detailed realworld example site
-- [Engineering Rust Web Applications](https://erwabook.com/intro/): A book describing full-stack Rust web-development, using Seed for the front-end
+- [Seed Realworld](https://github.com/seed-rs/seed-rs-realworld): A detailed realworld example
+- [Engineering Rust Web Applications](https://erwabook.com/intro/): A book describing full-stack Rust web development. It uses Seed!
```

Comment:
Both are outdated. I'm not sure if we want to mention it or remove the links or don't change it at all. Your choice.